tests/virt_autotest/install_package.pm

```diff
@@ -42,6 +42,26 @@ sub install_package {
         zypper_call("in -t pattern kvm_server kvm_tools", timeout => 300);
     }
 
+    #Solving the problem of missing perl-Clone package during installing qa_lib_virtauto package
+    if (is_sle('=12-SP5')) {
+        if (is_s390x) {
+            lpar_cmd("zypper --non-interactive rr update-repo");
+            lpar_cmd("zypper --gpg-auto-import-keys --non-interactive ar -f http://dist.nue.suse.com/ibs/QA:/SLE12SP5/update/ update-repo");
+	}
+        else {
+            zypper_call("--gpg-auto-import-keys --non-interactive ar -f http://dist.nue.suse.com/ibs/QA:/SLE12SP5/update/ update-repo", 1800);
+        }
+    }
+    else {
+        if (is_s390x) {
+            lpar_cmd("zypper --non-interactive rr update-repo");
+            lpar_cmd("zypper --gpg-auto-import-keys --non-interactive ar -f http://dist.nue.suse.com/ibs/QA:/SLE15SP6/update/ update-repo");
+        }
+        else {
+            zypper_call("--gpg-auto-import-keys --non-interactive ar -f http://dist.nue.suse.com/ibs/QA:/SLE15SP6/update/ update-repo", 1800);
+        }
+    }
+
```

Comment:
@alice-suse This is related to my email - https://suse.slack.com/files/USLACKBOT/F07U43MSRCM/_action_required__review_of_changes_needed_for_common_criteria_in_the_next_few_days

No need to do anything else other than the ticket, `QA` is already being synced.

Reply:
@foursixnine This is a different thing. This PR is about adding aother repo than QA Head repo(https://download.suse.de/ibs/QA:/Head/) to fulfill test pkg installation dependency.

Reply:
@alice-suse I see that now, however, `QA:SLE`, is already being mirrored, in https://mirror.nue2.suse.org/ibs/, because I asked BuildOPS to add `QA` repo, so we'd have it also available on NUE, because I knew only about `QA`; hence me asking to have tickets with the multiple assets needed, so we understand what's needed and can coordinate better, because i.e [variable expansion](https://progress.opensuse.org/issues/169159)  is needed before fully having [proper mirror support](https://progress.opensuse.org/issues/168097#note-16)

Reply:
@foursixnine Yes, that will be helpful if you can coordinate the needed images/repos in DE local mirror. In VT squad, @waynechen55 is working within https://progress.opensuse.org/issues/168352(and maybe new sub-tickets under the parent ticket) for test resource replacement. I will let you guys further communicate on this topic. Let me know if my involvement is needed :) .

Reply:
@alice-suse Did I tell you that its always a pleasure to work with you? :D I'll ping you if anything!